configure.ac

```diff
@@ -14,19 +17,49 @@ AS_IF([test "x$with_systemdsystemunitdir" != "xno"], [
   AC_SUBST([systemdsystemunitdir], [$with_systemdsystemunitdir])
 ])
 
+# build updater with metrics
+AC_ARG_ENABLE([metrics],
+              AS_HELP_STRING([--disable-metrics], [Whether to build eos-updater with metrics support]),
+              [want_metrics=$withval],
+              [want_metrics=yes])
+AM_CONDITIONAL([WITH_METRICS], [test "x$want_metrics" = 'xyes'])
+
+# port used for eos-server
+AC_ARG_WITH([server-port],
+            AS_HELP_STRING([--with-server-port], [Port number for the update server]),
+            [port=$withval],
+            [port=12345])
```

Comment:
>
> If you make it runtime configurable, then you'll have to listen for configuration changes and recreate the avahi service file and systemd service and socket unit files. It gets a bit out of hand, so I was wondering if it is actually eos-updater who should be responsible for that.
>
> I'll try to elaborate a bit more about it in the wrap up document.
>

Oh yuck, okay that sounds quite non-trivial. Probably leave it in the
wrap-up document. I was thinking that having a hardcoded port could cause
problems if you're testing multiple builds on the build server but it looks
like this is mostly unavoidable.